dali/operators/sequence/optical_flow/turing_of/optical_flow_turing.cc

```diff
@@ -152,6 +152,7 @@ void OpticalFlowTuring::CalcOpticalFlow(
 
 
 void OpticalFlowTuring::SetInitParams(dali::optical_flow::OpticalFlowParams api_params) {
+  init_params_ = {};
```

Comment:
Can we move it to optical_flow_turing.h ?

Reply:
I can put it _also_ there, but I think that resetting the parameters here has its value.

Reply:
Up to you.